src/content/developers/docs/smart-contracts/testing/index.md

```diff
@@ -1,37 +1,223 @@
+
+## Testing tools and libraries {#testing tools-and-libraries}
+
+### Unit testing tools {#unit-testing-tools}
+
+**Solidity-Coverage** - _A Solidity code coverage tool useful for testing smart contracts._
 - [GitHub](https://github.com/sc-forks/solidity-coverage)
 
-**Whiteblock Genesis -** **_An end-to-end development sandbox and testing platform for blockchain._**
+**Waffle** - _A framework for advanced smart contract development and testing (based on ethers.js)_.
+- [Documentation](https://ethereum-waffle.readthedocs.io/en/latest/)
+- [GitHub](https://github.com/TrueFiEng/Waffle) 
+- [Website](https://getwaffle.io/)
+
+**Remix Tests** - _A tool for testing Solidity smart contracts. Works underneath Remix IDE "Solidity Unit Testing" plugin which is used to write and run test cases for a contract._
+- [Documentation](https://remix-ide.readthedocs.io/en/latest/unittesting.html) 
+- [GitHub](https://github.com/ethereum/remix-project/tree/master/libs/remix-tests)  
+
+**OpenZeppelin Test Helpers** - _Assertion library for Ethereum smart contract testing. Make sure your contracts behave as expected!_
+- [GitHub](https://github.com/OpenZeppelin/openzeppelin-test-helpers) 
+- [Documentation](https://docs.openzeppelin.com/test-helpers) 
+
+**OpenZeppelin test environment** - _Highly configurable environment for blazing-fast smart contract testing._
+- [GitHub](https://github.com/OpenZeppelin/openzeppelin-test-environment)
+- [Documentation](https://docs.openzeppelin.com/test-environment/) 
+
+**Whiteblock Genesis** - _An end-to-end development sandbox and testing platform for blockchain._
 
 - [Whiteblock.io](https://whiteblock.io)
 - [Documentation](https://docs.whiteblock.io)
 - [GitHub](https://github.com/whiteblock/genesis)
 
-**OpenZeppelin Test Environment -** **_Blazing fast smart contract testing. One-line setup for an awesome testing experience._**
+**Truffle smart contract test framework** - _Automated testing framework to make testing your contracts a breeze._
+- [Documentation](https://trufflesuite.com/docs/truffle/testing/testing-your-contracts/) 
+- [Website](https://trufflesuite.com/) 
 
-- [GitHub](https://github.com/OpenZeppelin/openzeppelin-test-environment)
-- [Documentation](https://docs.openzeppelin.com/test-environment/)
+**Brownie unit testing framework** - _Brownie utilizes Pytest, a feature-rich test framework that lets you write small tests with minimal code, scales well for large projects, and is highly extendable._
+- [Documentation](https://eth-brownie.readthedocs.io/en/v1.0.0_a/tests.html) 
+- [GitHub](https://github.com/eth-brownie/brownie) 
+
+### Static analysis tools {#static-analysis-tools}
+
+**Oyente** - _Static analysis tool for analyzing vulnerabilities in EVM bytecode with symbolic execution._
```

Comment:
Same here. The website seems down and the last commit was 2 years ago. Wonder if this is still used by the devs/community since I prefer to mention a couple and well-known tools than a big list where new developers can get lost.

Reply:
Yeah, I agree with that. Someone else mentioned the OpenZeppelin Test Helpers tool was deprecated in a [recent issue](https://github.com/ethereum/ethereum-org-website/issues/6551). I'll take out both tools now.